.ci/jobs.t/elastic+elasticsearch+periodic+dra-snapshots.yml

```diff
@@ -0,0 +1,45 @@
+---
+- job:
+    name: elastic+elasticsearch+%BRANCH%+periodic+dra-snapshot
+    workspace: /dev/shm/elastic+elasticsearch+%BRANCH%+periodic+dra-snapshot
+    display-name: "elastic / elasticsearch # %BRANCH% - DRA snapshot"
+    description: "Publishing Daily Releasable Artifacts (DRAs) of Elasticsearch %BRANCH% snapshots.\n"
+    builders:
+      - inject:
+          properties-file: '.ci/java-versions.properties'
+          properties-content: |
+            JAVA_HOME=$HOME/.java/$ES_BUILD_JAVA
+            RUNTIME_JAVA_HOME=$HOME/.java/$ES_RUNTIME_JAVA
+      - shell: |
+            #!/usr/local/bin/runbld --redirect-stderr
+            DRA_VERSION=`cat build-tools-internal/version.properties \
+                  | grep elasticsearch \
+                  | sed "s/elasticsearch//g" \
+                  | sed "s/ //g" \
+                  | sed "s/=//"`
+            beatsBuildId=`./.ci/scripts/resolve-dra-manifest.sh beats master`
+            mlCppBuildId=`./.ci/scripts/resolve-dra-manifest.sh ml-cpp master`
+            set -euo pipefail
+            set +x
+            VAULT_TOKEN=$(vault write -field=token auth/approle/login role_id=$VAULT_ROLE_ID secret_id=$VAULT_SECRET_ID)
+            export VAULT_TOKEN
+            $WORKSPACE/.ci/scripts/run-gradle.sh assemble -Ddra.artifacts=true -Ddra.artifacts.dependency.${beatsBuildId} -Ddra.artifacts.dependency.${mlCppBuildId}
+            unset VAULT_TOKEN
+            set -x
+            # Artifacts should be generated
+            docker run --rm \
+                   --name release-manager \
+                   -e VAULT_ADDR \
+                   -e VAULT_ROLE_ID \
+                   -e VAULT_SECRET_ID \
+                   --mount type=bind,readonly=false,src="$PWD",target=/artifacts \
+                   docker.elastic.co/infra/release-manager:latest \
+                   cli collect \
+                   --project elasticsearch \
+                   --branch "master" \
+                   --commit `git rev-parse HEAD` \
```

Comment:
We can just use the `GIT_COMMIT` env var here provided by Jenkins.

Reply:
fixed